### ISSUE TITLE
[Issue]: Albums split by "Digital Media XX" appear as individual albums instead of merged (inline with "Disk XX" or "CD XX")

### ISSUE THREAD

**Issue by othyn** [jellyfin/jellyfin#11211]
### Please describe your bug

It appears #8713 was merged in 2022 to fix this, but it appears it doesn't work, with comments sprawled across the Discord channel also having the same problem.

The current workaround is to use a `CD` or `Disk` prefix to the medium format instead of `Digital Media`. As the common use case for this is Lidarr to automate the library management process, its a change to its Media Management > Multi Disc Track Format to instead force it to be `.../CD {medium:00}/...` instead of `.../{Medium Format} {medium:00}/...`. However, if things are being managed properly then ideally `Digital Media` should be an accepted prefix to split an album.

I'm not sure why #8713 doesn't work, but my gut feeling is the whitespace perhaps needs to be escaped? I'm not a C# dev, but its a common one on the list of gut feelings when dealing with strings.

I can confirm other albums split by `CD XX` and `Disk XX` are working as intended.

Example of the split albums not being merged:

<img width="566" alt="Screenshot 2024-03-25 at 11 07 23" src="https://github.com/jellyfin/jellyfin/assets/7256684/a390b9a7-fad0-4e81-bfb3-fdda94efabba">

With the path being shown:

<img width="871" alt="Screenshot 2024-03-25 at 11 35 01" src="https://github.com/jellyfin/jellyfin/assets/7256684/098682b8-e750-4c8c-9562-c7f3cb73bbaf">

### Reproduction Steps

1. Have music in a media folder that is organised by Lidarr as:
```
- Artist
    - Album
        - Digital Media 01
            - Track 01
            - Track 02
            - ...
        - Digital Media 02
            - Track 01
            - Track 02
            - ...
        - ...
```
2. Fresh Lidarr instance.
3. Mount and create a Music media library.
4. Let it scan and finish processing.
5. Navigate to the Albums tab of the Music library added above.
6. Any media that is split by `Digital Media` shows as individual albums instead of merged.

### Jellyfin Version

10.8.13

### Environment

```markdown
- OS: Linux
- Linux Kernel: -
- Virtualization: Docker
- Clients: Browser
- Browser: Arc `1.35.0` (`47750`) (Chromium Engine Version `123.0.6312.59`)
- FFmpeg Version: -
- Playback Method: Direct
- Hardware Acceleration: none
- GPU Model: none
- Plugins: none
- Reverse Proxy: Traefik
- Base URL: none
- Networking: Bridge
- Storage: local
```

### Jellyfin logs

```shell
None? Or I can be pointed in the right direction to source relevant logs for this bug from someone in-the-know on what they are looking for.
```

### FFmpeg logs

```shell
N/A
```

### Please attach any browser or client logs here

N/A

### Please attach any screenshots here

(in main post)

### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

**Comment by othyn** [comment]
It appears there aren't any tests to check for the addition in:
https://github.com/jellyfin/jellyfin/blob/775815d09ca4cb96783ecdcda14520f6d12c3385/tests/Jellyfin.Naming.Tests/Music/MultiDiscAlbumTests.cs#L7

I'll see if I can get a C# environment setup later and have a go at fixing the issue. Get it added as a condition to the test suite.

**Comment by gnattu** [comment]
This is not in 10.8 branch:
https://github.com/jellyfin/jellyfin/blob/e93d03d8cbff2122d7296f477604146f64758a73/Emby.Naming/Common/NamingOptions.cs#L175

Will be included in upcoming 10.9, please reopen a new issue if this still happens in 10.9.

**Comment by othyn** [comment]
Ah okay, I should have checked the 10.8 release branch first, my bad. Looking forward to the 10.9 release https://github.com/jellyfin/jellyfin/discussions/11051!
